### ISSUE TITLE
Sensors unavailable on Buderus TC100.2 (EASYCONTROL)

### ISSUE THREAD

**Issue by ChristophPech** [bosch-thermostat/home-assistant-bosch-custom-component#194]
**Describe the bug**
When using the integration I can change the temperature of my thermostats just fine, but all their sensors show as unavailable.

**Version**
  - HA version: 2022.9.2
  - HA Bosch component version: 0.18.3

**Debug SCAN**
https://gist.github.com/ChristophPech/f69b55c356217c946e7a8a1f7aee9162

**Error from Logfile**
```
2022-09-13 11:20:01.564 ERROR (MainThread) [homeassistant] Error doing job: Task exception was never retrieved
Traceback (most recent call last):
File "/config/custom_components/bosch/__init__.py", line 417, in thermostat_refresh
await self.component_update(SENSOR, event_time)
File "/config/custom_components/bosch/__init__.py", line 396, in component_update
await entity.bosch_object.update()
File "/usr/local/lib/python3.10/site-packages/bosch_thermostat_client/sensors/crawl.py", line 48, in update
self._data[key][RESULT] = process_result(result.get(VALUE))
File "/usr/local/lib/python3.10/site-packages/bosch_thermostat_client/sensors/crawl.py", line 37, in process_result
if len(result) == 1:
TypeError: object of type 'NoneType' has no len()
```

**Comment by ChristophPech** [comment]
**Screenshot of the Issue**

![Screenshot_20220913_115351](https://user-images.githubusercontent.com/3427525/189872881-07116982-6adf-4baa-83e7-e220e5e1fd69.png)

**Comment by pszafer** [comment]
Did you enable those sensors?

**Comment by ChristophPech** [comment]
Yes. All of them.

**Comment by ChristophPech** [comment]
I debugged it a little bit myself, and the error occurs when the sensors "rdevice10", "rdevice11", "rdevice12" and "rdevice13" are tried to be updated, but apparently they don't exist. 

I made a bad attempt at fixing it: https://github.com/ChristophPech/bosch-thermostat-client-python/blob/test/bosch_thermostat_client/sensors/crawl.py

But returning 0 will trigger some exception in while the unit conversion is going on. But at least that solves all my problems and all sensors show up correctly.

**Comment by pszafer** [comment]
better to do
```python
if result and len(result) == 1:
```

I'll fix it in next version

**Comment by pszafer** [comment]
Fixed in 0.19.2.
Basically `/devices/device1/` should be fetched one level deeper.
If you need those sensors for some reason feel free to open new issue.

**Comment by ChristophPech** [comment]
Thanks. It works, and I can see how your fix is better. Send a tip.
